consensus/e2e_tests/utils_test.go

```diff
@@ -251,13 +252,12 @@ func WaitForNetworkStateSyncEvents(
 		msg, err := codec.GetCodec().FromAny(anyMsg)
 		require.NoError(t, err)
 
-		stateSyncMessage, ok := msg.(*typesCons.StateSyncMessage)
+		_, ok := msg.(*typesCons.StateSyncMessage)
 		require.True(t, ok)
-
-		return stateSyncMessage.MsgType == msgType
+		// TODO: check state sync msg type
```

Comment:
Is there anything to check?

Reply:
I don't think anything is needed there, so I removed the comment